lib/matplotlib/axes/_base.py

```diff
@@ -2034,6 +2035,28 @@ def set_autoscaley_on(self, b):
         """
         self._autoscaleYon = b
 
+    @property
+    def use_sticky_edges(self):
+        """
+        When autoscaling, whether to obey all `Artist.sticky_edges`.
+
+        Default is ``True``.
+
+        Setting this to ``False`` ensures that the specified margins
+        will be applied, even if the plot includes an image, for
+        example, which would otherwise force a view limit to coincide
+        with its data limit.
+
+        The changing this property does not change the plot until
+        `autoscale` or `autoscale_view` is called.
+        """
+        return self._use_sticky_edges
+
+    @use_sticky_edges.setter
+    def use_sticky_edges(self, b):
+        self._use_sticky_edges = bool(b)
```

Comment:
I was wondering about that also, but decided against it.  I could be convinced otherwise for now, but long term I think we might want to consolidate the autoscaling at draw time instead of having calls to `autoscale_view` sprinkled all over the place.  I don't think it makes sense to trigger autoscaling with every change in the state of the plot.